### PR TITLE
Feature/75 has x in adapter properties

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**

### CODE REVIEW

File: src/main/java/com/exasol/adapter/AdapterProperties.java [hunk 1]
```diff
@@ -180,21 +185,138 @@ public static boolean isRefreshingVirtualSchemaRequired(final Map<String, String
...
+     * Check if the log level property is set
+     *
+     * @return <code>true</code> if log level property is set
+     */
+    public boolean hasLogLevel() {
+        return containsKey(LOG_LEVEL_PROPERTY);
+    }
+
+    /**
+     * Check if the sql dialect property is set
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/AdapterProperties.java]: `     * Check if the SQL dialect property is set`

File: src/main/java/com/exasol/adapter/AdapterProperties.java [hunk 2]
```diff
@@ -180,21 +185,138 @@ public static boolean isRefreshingVirtualSchemaRequired(final Map<String, String
...
+     *
+     * @return <code>true</code> if sql dialect property is set
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/AdapterProperties.java]: `     * @return <code>true</code> if SQL dialect property is set`

File: src/main/java/com/exasol/adapter/AdapterProperties.java [hunk 3]
```diff
@@ -180,21 +185,138 @@ public static boolean isRefreshingVirtualSchemaRequired(final Map<String, String
...
+     */
+    public boolean hasSqlDialect() {
+        return containsKey(SQL_DIALECT_PROPERTY);
+    }
+
+    /**
+     * Check if the excluded capabilities property is set
+     *
+     * @return <code>true</code> if  excluded capabilities property is set
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/AdapterProperties.java]: double space

File: src/main/java/com/exasol/adapter/AdapterProperties.java [hunk 4]
```diff
@@ -180,21 +185,138 @@ public static boolean isRefreshingVirtualSchemaRequired(final Map<String, String
...
+     * Check if the ignore errors property is set
+     *
+     * @return <code>true</code> if ignore errors property is set
+     */
+    public boolean hasIgnoreErrors() {
+        return containsKey(IGNORE_ERRORS_PROPERTY);
+    }
+
+    /**
+     * Get empty map
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/AdapterProperties.java]: Get empty property list

File: src/main/java/com/exasol/adapter/AdapterProperties.java [hunk 5]
```diff
@@ -180,21 +185,138 @@ public static boolean isRefreshingVirtualSchemaRequired(final Map<String, String
...
+     *
+     * @return empty map
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/AdapterProperties.java]: empty properties

File: src/test/java/com/exasol/adapter/AdapterPropertiesTest.java [hunk 6]
```diff
@@ -83,4 +83,147 @@ void testIsLocalSourceTrue() {
...
+    }
+
+    @Test
+    void hasPasswordFalseByDefault() {
+        assertThat(AdapterProperties.emptyProperties().hasPassword(), equalTo(false));
+    }
+
+    @Test
+    void hasPassword() {
+        this.rawProperties.put(PASSWORD_PROPERTY, "user");
```
Comment (redcatbear) [inline src/test/java/com/exasol/adapter/AdapterPropertiesTest.java]: "secret"

File: src/main/java/com/exasol/adapter/AdapterProperties.java (hunk 1 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/AdapterProperties.java]: fixed

File: src/main/java/com/exasol/adapter/AdapterProperties.java (hunk 2 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/AdapterProperties.java]: fixed

File: src/main/java/com/exasol/adapter/AdapterProperties.java (hunk 3 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/AdapterProperties.java]: fixed

File: src/main/java/com/exasol/adapter/AdapterProperties.java (hunk 4 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/AdapterProperties.java]: fixed

File: src/main/java/com/exasol/adapter/AdapterProperties.java (hunk 5 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/AdapterProperties.java]: fixed

File: src/test/java/com/exasol/adapter/AdapterPropertiesTest.java (hunk 6 above)
Comment (AnastasiiaSergienko) [reply src/test/java/com/exasol/adapter/AdapterPropertiesTest.java]: fixed

**Pull request merged**